### PR TITLE
Add feedback/indicator when copying recovery phrase

### PR DESCRIPTION
**Pull request by muliswilliam**
Added functionality to indicate when a user copies recovery phrase by:
- Changing button text from 'Copy' to 'Copied' when  a user clicks on Copy button
- Adding a checkmark when text is copied

The button resets back to 'Copy' after 5 seconds.

Resolves https://github.com/brave/brave-browser/issues/19021

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

https://user-images.githubusercontent.com/48117473/145448595-35d9c595-04cd-401c-b23f-7144572ea124.mov

### CODE REVIEW

File: components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx [hunk 1]
```diff
@@ -50,7 +52,10 @@ function OnboardingBackup (props: Props) {
         )}
       </RecoveryPhraseContainer>
       <Tooltip text={getLocale('braveWalletToolTipCopyToClipboard')}>
-        <CopyButton onClick={onCopy}>{getLocale('braveWalletButtonCopy')}</CopyButton>
+        <CopyButton onClick={onCopy}>
+          {recoverPhraseCopied && <BigCheckMark />}
+          {recoverPhraseCopied ? 'Copied' : getLocale('braveWalletButtonCopy')}
```
Comment (Douglashdaniel) [inline components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx:57]: String needs to be added here `getLocale('braveWalletButtonCopied')` 😎

Comment (muliswilliam) [reply components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx:57]: thank you, changing 👌

**Review by Douglashdaniel — APPROVED** [review]
Looks good!

File: components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx [hunk 2]
```diff
@@ -25,11 +26,12 @@ export interface Props {
   isRecoveryTermsAccepted: boolean
   onSubmitTerms: (key: string, selected: boolean) => void
   recoverPhrase: string[]
+  recoverPhraseCopied: boolean
```
Comment (petemill) [inline components/brave_wallet_ui/components/desktop/wallet-onboarding/recovery/index.tsx:29]: `isRecoverPhraseCopied` seems to fit more with existing pattern for booleans

File: components/brave_wallet_ui/stories/screens/backup-wallet.tsx [hunk 3]
```diff
@@ -87,8 +90,19 @@ function BackupWallet (props: Props) {
 
   const onCopyToClipboard = async () => {
     await copyToClipboard(recoveryPhrase.join(' '))
+    setRecoverPhraseCopied(true)
   }
 
+  React.useEffect(() => {
+    const timer = setTimeout(() => {
```
Comment (petemill) [inline components/brave_wallet_ui/stories/screens/backup-wallet.tsx:97]: nit: `if !recoverPhraseCopied` then don't need to set timeout

File: components/resources/wallet_strings.grdp [hunk 4]
```diff
@@ -84,6 +84,7 @@
   <message name="IDS_BRAVE_WALLET_SWAP_FEES_NOTICE" desc="Swap fees notice">Quote includes a 0.875% Brave fee.</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_CONTINUE" desc="General Continue button">Continue</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_COPY" desc="General Copy button">Copy</message>
+  <message name="IDS_BRAVE_WALLET_BUTTON_COPIED" desc="General Copied button">Copied</message>
```
Comment (petemill) [inline components/resources/wallet_strings.grdp:87]: Perhaps a more useful description for translators would be "Confirmation that text has been copied to clipboard"

**Review by petemill — COMMENTED** [review]
Just some minor nits

**Review by petemill — APPROVED** [review]

**Pull request merged**